app/views/layouts/application.html.erb

```diff
@@ -1,17 +1,23 @@
 <!DOCTYPE html>
-<html>
+<htm>
```

Comment:
any specific reason why this isn't "html"?

Reply:
haha no.. changed (: